wkcuber/wkcuber/cubing.py

```diff
@@ -319,25 +319,28 @@ def cubing(
 
     logging.info("Found source files: count={} size={}x{}".format(num_z, num_x, num_y))
 
-    ensure_wkw(target_wkw_info)
-
     with get_executor_for_args(executor_args) as executor:
         job_args = []
         # We iterate over all z sections
         for z in range(start_z, num_z + start_z, BLOCK_LEN):
             # Prepare z batches
```

Comment:
If I see this correctly you don’t prepare z batches anymore, but rather pass the views directly?  Maybe the comments can be adapted

Reply:
I have updated the comments